dictionary/Dictionary.js

```diff
@@ -45,6 +45,14 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the localization phrase exists at dictionary
```

Comment:
I think it's "in dictionary", also, please, terminate the sentence using a period.

It might also be better to use the wording "the specified localization phrase" to narrow down the context.